_jobs/positions/__QA-testing-engineer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: QA Testing & Automation Engineer
+is_fellow: true
+lever_link:
+position: 1
+lede_markdown: We’re looking for skilled manager and mentor to lead engineering practices for City of Austin’s Design, Technology, and Innovation Fellows program.
+---
+
+## Overview
+We are looking for an experienced, skilled, and passionate QA Testing & Automation professional to help streamline our QA process for current and future projects. The person in this role will ensure timely and frequent release schedules without compromising on quality and accessibility for Austin residents.
+
+This vital position has the potential to create positive change inside our organization and across City of Austin services. The person in this role will be instrumental in shaping the future of digital, accessible, and equitable city services. We're looking for someone passionate, creative, resourceful, civic-minded, and impact-driven. This may involve a wide range of activities, from helping solidify existing QA, testing, and release protocols to helping write or review unit tests and set up cross-browser testing.
+
+
+## Salary
+
+$75,000-$86,000, commensurate with experience
+
+## What you’ll do:
+
+* Take the lead in collaborating to build and maintain effective testing practices when appropriate, including:
+  * Designing and implementing unit tests. 
```

Comment:
Interested in y'alls opinion on this one. While it would be great to have someone in QA that could write unit tests, i think there is a chance that the best candidates for many of these tasks won't have that on their radar. Whether developers write unit tests or the QA person does it seems like should be up for debate. Also, if its on there and a great QA automation engineer that is female reviews this, they are more likely not to apply.

I recommend something more generic like "Design and implement QA strategies that ensure automation for unit testing, integration testing, etc are used appropriately to ensure high quality and efficient applications."??? Thoughts?

Reply:
Eric and I talked about this. Given the size of our dev team and our bandwidth limitations, we really need a QA person who can write unit tests. But we're hopeful that a coding requirement wouldn't limit our candidate pool significantly. From Eric's and my experience, it isn't unusual for a QA engineer to also write unit tests.